h3-iroh/examples/server.rs

```diff
@@ -99,10 +103,11 @@ async fn handle_connection(incoming: Incoming, root: Arc<Option<PathBuf>>) -> Re
             }
             Err(err) => {
                 error!("accept error: {err:#}");
-                match err.get_error_level() {
-                    ErrorLevel::ConnectionError => break,
-                    ErrorLevel::StreamError => continue,
-                }
+                // TODO:
+                // match err {
+                //     ErrorLevel::ConnectionError => break,
+                //     ErrorLevel::StreamError => continue,
+                // }
```

Comment:
I've not looked up the changes in h3.  But from the API it looks like this is the ConnectionError so should be a break.  While `req_resolver.resolve_request().await?` now surfaces the stream error.